spec/models/gogovan_order_spec.rb

```diff
@@ -14,13 +14,12 @@
     it{ is_expected.to have_db_column(:driver_license).of_type(:string)}
   end
 
-  describe 'save_booking' do
+  describe 'update_booking' do
     let(:booking_id) { rand(1000000..9999999) }
-    it 'should create record with booking_id' do
-      expect{
-        GogovanOrder.save_booking(booking_id)
-      }.to change(GogovanOrder, :count).by(1)
-      expect(GogovanOrder.last.booking_id).to eq(booking_id)
+    let(:ggv_order) { create :gogovan_order }
+    it 'should update record with booking_id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.